mtenn/conversion_utils/visnet.py

```diff
@@ -0,0 +1,237 @@
+"""
+Representation and strategy for ViSNet model.
+"""
+from copy import deepcopy
+import torch
+from torch.autograd import grad
+from torch_geometric.utils import scatter
+
+from mtenn.model import GroupedModel, Model
+from mtenn.strategy import ComplexOnlyStrategy, ConcatStrategy, DeltaStrategy
+
+HAS_VISNET_FLAG = False
```

Comment:
I would just make this variable called `HAS_VISNET` rather than `HAS_VISNET_FLAG`